README.md

```diff
@@ -79,6 +79,9 @@ We now have actions for database deployments
 
 Refer to [starter templates](https://github.com/Azure/actions-workflow-samples/tree/master/Database) for examples.
 
+## Run Azure Policy Compliance Scans

```

Comment:
Shouldn't the area be called just `Azure Policy Compliance`

Reply:
Ideally it should be along the lines of 'Azure Policy integrations'. 
Since we have only 1 action, have named it like this. Will update when we add more policy actions